application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -483,6 +483,27 @@
 
 .count {
   color: $theme.textSecondaryColor;
+}
+@media (min-width: 768px) {
+  .inline-input-row>.col-sm-4>dl {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+  }
+
+  .inline-input-row>.col-sm-4 {
+    float: none;
+    height: 100%;
+    display: table-cell;
+    vertical-align: top;
```

Comment:
You don't need these, I think, if you use ``display:flex`` on the row (with ``flex-direction:row`` of course).

Reply:
I wanted all the cells/columns to have the same height. It didn't seem to work any other way. I tried using this but it doesn't act as wished.

Reply:
> I wanted all the cells/columns to have the same height.

I just had to set ``height:100%`` on the 3 definition lists. Here's what I tried:

![xpoll-flex-styles](https://user-images.githubusercontent.com/652632/137276690-a8cadc00-27bf-4490-9d63-700cd5678f31.png)

I only added styles to the ``div.row`` and the 3 definition lists (because I used the inspector). It seemed to work fine, at least in Firefox.